### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell`

### CODE REVIEW

File: rtgui/batchqueuepanel.cc [hunk 1]
```diff
@@ -192,7 +192,7 @@ void BatchQueuePanel::init (RTWindow *parent)
     saveFormatPanel->init (options.saveFormatBatch);
 }
 
-// it is expected to have a non null forceOrientation value on Preferences update only. In this case, qsize is ingored and computed automatically
```
Comment (Floessie) [inline rtgui/batchqueuepanel.cc:195]: @heckflosse This is my favorite! 🤣

Comment (heckflosse) [reply rtgui/batchqueuepanel.cc:195]: @Floessie I really don't understand :(

File: rtengine/improcfun.cc [hunk 2]
```diff
@@ -5785,7 +5785,7 @@ void ImProcFunctions::chromiLuminanceCurve (PipetteBuffer *pipetteBuffer, int pW
                     Lprov1 = l_r * 100.f;
 
                     float Chprov2 = sqrt (SQR (atmp) + SQR (btmp)) / 327.68f;
-                    //Gamut control especialy fot negative values slightly different of gamutlchonly
+                    //Gamut control especially fot negative values slightly different of gamutlchonly
```
Comment (Beep6581) [inline rtengine/improcfun.cc]: `s/fot/for/`

Comment (Beep6581) [inline rtengine/improcfun.cc]: `s/different of/different from/`

File: rtengine/dcraw.c [hunk 3]
```diff
@@ -3826,7 +3826,7 @@ void CLASS remove_zeroes()
 }
 
 /*
-   Seach from the current directory up to the root looking for
+   Search from the current directory up to the root looking for
```
Comment (Beep6581) [inline rtengine/dcraw.c]: @heckflosse what do we do about the changes to the dcraw* files?
Am I right that the changes to `dcraw.c` should be reverted, but the changes to `dcraw.cc` and to `dcraw.patch` can stay?

File: rtgui/batchqueuepanel.cc (hunk 1 above)
Comment (agriggio) [reply rtgui/batchqueuepanel.cc:195]: @heckflosse seems your eyes are ingoring the typo ;-)

Comment (Floessie) [reply rtgui/batchqueuepanel.cc:195]: 😂

**Comment by heckflosse** [comment]
@Beep6581 changes to dcraw.c and dcraw.patch should be reverted. dcraw.cc can stay

**Comment by luzpaz** [comment]
Made requested revisions. Please review them in 8f78e27f5712d5841e79a34a52fd0c99c3ba750f

**Comment by heckflosse** [comment]
@luzpaz Please no corrections for `dcraw.c` and `dcraw.patch`. Corrections for `dcraw.cc` are ok.

`dcraw.c` is the original dcraw code from Dave Coffin and should stay unmodifed.
`dcraw.patch` is an auto generated file which does not need to get typo corrections.
Imho, we could remove dcraw.patch from repository. @Beep6581 @Floessie @agriggio ?

**Comment by luzpaz** [comment]
@heckflosse `dcraw.c` and `dcraw.patch` have been reverted. The commits in 8f78e27 look like I'm modifying them but they are actually reverting the changes in 21135055ed9a07b1d0bd8285b5633e8ef8bc259e
If you want, I can squash all the commits down and push -f for more clarity ?

**Comment by heckflosse** [comment]
@luzpaz 

> but they are actually reverting the changes

Ah, ok, missed that. Then it's fine. 👍  from me to merge
Any objections from other devs?

**Comment by Beep6581** [comment]
@luzpaz +1 for squash.
@heckflosse +1 for dropping the patch.

**Comment by luzpaz** [comment]
> +1 for squash.  
> +1 for dropping the patch  

Done, re-pushed. 
Edit: I had a typo in the git summary. Fixed and repushed.

**Comment by heckflosse** [comment]
@luzpaz Thank you very much 👍

**Pull request merged**